core/src/main/java/hudson/Functions.java

```diff
@@ -469,6 +469,18 @@ public static Map getSystemProperties() {
         return new TreeMap<Object,Object>(System.getProperties());
     }
 
+    /**
+     * Gets the system property indicated by the specified key.
+     * 
+     * Delegates to {@link SystemProperties#getString(java.lang.String)}.
+     * 
+     * @since TODO
```

Comment:
Generally it's not required for restricted methods.
Especially in this case, I suspect the method has been removed at some point